content/english/anti-vegan/humans-are-superior.md

```diff
@@ -0,0 +1,107 @@
+---
+title: "Humans Are Superior"
+date: 2019-10-12T07:28:15+13:00
+name: tery-hardwicke
+contributors:
+    - 
+        name: tery-hardwicke
+        role: author
+    - 
+        name: sam-martin
+        role: editor
+draft: false
+description: "How to respond to the argument: We are entitled to use animals as a resource because humans are superior to nonhuman animals"
+weight: 5
+---
+# Argument
+
+> ...but, humans are superior...  
+>   
+> We are entitled to use animals as a resource because humans are superior to nonhuman animals
+
+# Rebuttal
+
+## Quick Responses
+
+> Superiority supposes an arbitrary line that falsely identifies one group as deservedly superior to another.   
```

Comment:
'supposes an arbitrary line'. I didn't completely understand the meaning, apologies, had to look up arbitrary! I appreciate the explanation directly below does clarify but I'm concerned people may get lost. Also, I myself would not actually use this sentence as a response as I would never remember it because it's using words I don't normally use when speaking on a daily basis. Could there be a way of simplifying it, what do you think?

Reply:
You need some new words :)  Nah, just kidding, I forget how differently philosophers speak... looking for alternatives....

Reply:
Much better! I do indeed need new words...I'm trying.